### DISCUSSION TITLE
Errors from and questions about ReconstructMesh, DensifyPointCloud

### DISCUSSION THREAD

**Discussion by nimafo** [cdcseacave/openMVS#876] [Q&A] [answered]
Hi, 
I am reproducing the codes in the very interesting tutorial post on [COLMAP - OpenMVS](https://github.com/cdcseacave/openMVS/issues/692). In the mesh reconstruction step using`ReconstructMesh`, I get the following errors: 

> 18:17:22 [App     ] error: failed loading image header
18:17:22 [App     ] error: failed reloading image '/roo18:17:22 [App     ] error: failed reloading image '/root/00_codes/PG_LM/COLMAP_project_folder/Paper_workflow/Exmple1/Output from COLMAP_PINHOLE MODEL/txt/images/DSC_6226.JPG'

You can find the complete log [here](https://www.dropbox.com/s/ayuiytynab4aevr/ReconstructMesh-2210061817188B4897.log?dl=1). 

I run the following command: 
`ReconstructMesh --input-file "/openMVS_working_Dir/model_dense.mvs" --output-file "openMVS_working_Dir/model_dense_mesh.mvs"`

you can find the input I used here: [model_dense.mvs](https://www.dropbox.com/s/l3ydzvz942gv0gr/model_colmap_dense.mvs?dl=1)

previous steps: 
- output from InterfaceCOLMAP: [model_colmap.mvs]( https://www.dropbox.com/s/3sd5w4zaogdmyno/model_colmap.mvs?dl=1)
- output from `DensifyPointCloud`:[model_colmap_dense.mvs](https://www.dropbox.com/s/l3ydzvz942gv0gr/model_colmap_dense.mvs?dl=1) and [model_colmap_dense.ply](https://www.dropbox.com/s/4xf48d6eh53tw49/model_colmap_dense.ply?dl=1)

**My questions are**:
1. Why does the error I mentioned above happen when running `ReconstructMesh`? how can I solve it?
2. Should I expect a higher number of points after running  `DensifyPointCloud`?

**Comment by cdcseacave** [comment, marked as answer]
Does the image in the error actually exist on disk?

**Reply by nimafo** [reply]
You were right, the error is self-explaining. 
I repeated the process from scratch and it was resolved, sorry for this!
